examples/fetch/src/index.ts

```diff
@@ -8,7 +8,31 @@ const fetchMachine = createMachine({
         FETCH: 'loading'
       }
     },
-    loading: {}
+    loading: {
+      invoke: {
+        id: 'dogFetch',
+        src: (context, event) =>
+          fetch('https://dog.ceo/api/breeds/image/random').then((response) =>
+            response.json()
+          ),
+        onDone: {
+          target: 'success'
+        },
+        onError: {
+          target: 'failure'
+        }
```

Comment:
Other transitions here are using shortcut transition configs, so maybe good to unify those with that style?
```suggestion
        onDone: 'success',
        onError: 'failure'
```

Reply:
I'm thinking about saving it to `context`, so we'll keep it as an object